tests/integration/defs/perf/disagg/utils/artifactory_uploader.py

```
@@ -0,0 +1,297 @@
+        
+        # Determine archive extension and mode
+        compression_modes = {
+            'gz': ('tar.gz', 'w:gz'),
+            'bz2': ('tar.bz2', 'w:bz2'),
+            'xz': ('tar.xz', 'w:xz'),
+            '': ('tar', 'w')
+        }
+        
+        if compression not in compression_modes:
+            print(f"Unsupported compression type: {compression}")
+            print(f"   Supported: gz, bz2, xz, '' (no compression)")
```

Ruff (0.15.0)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)